ssp

```diff
@@ -6252,11 +6035,12 @@ sub check_for_usr_local_include_jpeglib_h {
 }
 
 sub check_for_bw_module_and_more_than_1024_vhosts {
+    return unless i_am_one_of( 'ea4', 'ea3' );
     my $modules = get_apache_modules_href();
     return unless defined $modules->{'bw_module'};
 
     # Remove on EA3 deprecation
-    my $httpdconf = $EA4 ? '/etc/apache2/conf/httpd.conf' : '/usr/local/apache/conf/httpd.conf';
+    my $httpdconf = i_am('ea4') ? '/etc/apache2/conf/httpd.conf' : '/usr/local/apache/conf/httpd.conf';
```

Comment:
Is this needed?  These should be symlinked so we could really just always say /usr/local/apache/conf/httpd.conf as if that's not an actual file it's going to be a symlink to the actual file. 

Def not a show stopper - I am more or less checking my logic there.

Reply:
We should never rely on a legacy symlink to exist for that purpose.  It already doesn't always exist on some production servers, already, so trying to rely on it would definitely break SSP in various ways.